disperser/controller/controller_metrics.go

```diff
@@ -264,142 +247,154 @@ func newDispatcherMetrics(
 		},
-		signingRateLabels,
+		[]string{"quorum"},
 	)
 
-	signingLatency := promauto.With(registry).NewSummaryVec(
-		prometheus.SummaryOpts{
-			Namespace:  controllerNamespace,
-			Name:       "validator_signing_latency_seconds",
-			Help:       "Latency for validators to sign batches",
-			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
+	globalSignedByteCount := promauto.With(registry).NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: controllerNamespace,
+			Name:      "global_signed_byte_count",
+			Help:      "Total number of bytes successfully signed by a critical mass of validators",
 		},
-		signingRateLabels,
+		[]string{"quorum"},
 	)
 
-	return &dispatcherMetrics{
-		sendChunksRetryCount:         sendChunksRetryCount,
-		processSigningMessageLatency: processSigningMessageLatency,
-		signingMessageChannelLatency: signingMessageChannelLatency,
-		attestationUpdateLatency:     attestationUpdateLatency,
-		attestationBuildingLatency:   attestationBuildingLatency,
-		thresholdSignedToDoneLatency: thresholdSignedToDoneLatency,
-		aggregateSignaturesLatency:   aggregateSignaturesLatency,
-		putAttestationLatency:        putAttestationLatency,
-		attestationUpdateCount:       attestationUpdateCount,
-		updateBatchStatusLatency:     updateBatchStatusLatency,
-		blobE2EDispersalLatency:      blobE2EDispersalLatency,
-		completedBlobs:               completedBlobs,
-		attestation:                  attestation,
-		blobSetSize:                  blobSetSize,
-		batchStageTimer:              batchStageTimer,
-		sendToValidatorStageTimer:    sendToValidatorStageTimer,
-		importantSigningThresholds:   importantSigningThresholds,
-		signatureThresholds:          batchSigningThresholdCount,
-		signedBatchCount:             signedBatchCount,
-		signedByteCount:              signedByteCount,
-		unsignedBatchCount:           unsignedBatchCount,
-		unsignedByteCount:            unsignedByteCount,
-		timeoutBatchCount:            timeoutBatchCount,
-		timeoutByteCount:             timeoutByteCount,
-		signingLatency:               signingLatency,
-	}, nil
-}
+	globalUnsignedByteCount := promauto.With(registry).NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: controllerNamespace,
+			Name:      "global_unsigned_byte_count",
+			Help:      "Total number of bytes that were not signed by a critical mass of validators",
+		},
+		[]string{"quorum"},
+	)
 
-func (m *dispatcherMetrics) reportSendChunksRetryCount(retries float64) {
-	if m == nil {
-		return
-	}
-	m.sendChunksRetryCount.WithLabelValues().Set(retries)
+	globalSigningFractionHistogram := promauto.With(registry).NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: controllerNamespace,
+			Name:      "global_signing_fraction_histogram",
+			Help:      "Histogram of the fraction of validators that signed each batch",
+			Buckets:   prometheus.LinearBuckets(0.0, 0.05, 21),
```

Comment:
Are linear buckets actually the best choice here? Would something like this make more sense?

`[]float64{0.0, 0.30, 0.55, 0.60, 0.65, 0.70, 0.75, 0.80, 0.85, 0.90, 0.95, 0.99, 1.0}`

It seems additional granularity around the top of the range could be helpful.

Reply:
We should discuss this. My grug instincts tingle when I think about trying to be fancy about buckets.

Reply:
I'm fine leaving this as linear buckets, just wanted to bring it up 👍